test/evergreen.yml

```diff
@@ -3743,6 +3743,44 @@ buildvariants:
     - name: ".stress-test-4"
     - name: format-abort-recovery-stress-test
 
+- name: ubuntu2004-stress-tests-arm64
+  display_name: Ubuntu 20.04 Stress tests (ARM64)
+  run_on:
+  # FIXME-WT-8946: Use a larger machine with a reduced number of parallel jobs to prevent OOM killer
+  # from interrupting the format sanitizer stress tests. Once we have root caused and fixed the
+  # issue, we should reduce to a smaller machine with the same number of parallel jobs as the cores
+  # available.
+  - ubuntu2004-arm64-large
+  expansions:
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` / 2" | bc)
+    make_command: ninja
+    test_env_vars:
+      LD_PRELOAD=/usr/lib/aarch64-linux-gnu/libeatmydata.so
+      PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
+    posix_configure_flags:
```

Comment:
From what I can tell, this doesn't get threaded through to the actual compile command. It ends up with either "compile wiredtiger address sanitizer" or "compile wiredtiger with builtins", neither of which expand posix_configure_flags.

I suspect that HAVE_DIAGNOSTIC is going to make it unlikely that we'll see the bug we are currently chasing - it enables diagnostic yields, which generally mean a barrier is used.

Reply:
It would also be good if we could get some additional flags into the build, so it more closely matches MongoDB. The flags are:        -moutline-atomics -DHAVE_ARM_NEON_INTRIN_H

Reply:
Will be addressed by WT-8958